source/guides/clear/telemetrics.rst

```diff
@@ -720,88 +733,109 @@ Reference
 The telemetry API
 =================
 
-Installing the :command:`telemetrics` bundle includes the libtelemetry C library,
-which exposes an API used by the telemprobd and telempostd daemons. You can
-use these in your applications as well. The API documentation is found in the
-:file:`telemetry.h` file in `Telemetrics client`_ repository.
+Installing the :command:`telemetrics` bundle includes the libtelemetry C
+library, which exposes an API used by the telemprobd and telempostd daemons.
+You can use these in your applications as well. The API documentation is found
+in the :file:`telemetry.h` file in `Telemetrics client`_ repository.
 
 Client configuration
 ====================
 
 The telemetry client will look for the configuration file located at
 :file:`/etc/telemetrics/telemetrics.conf` and use it if it exists. If the
-file does not exist, the client will use the default configuration located
-at :file:`/usr/share/defaults telemetrics/telemetrics.conf`. To modify or
+file does not exist, the client will use the default configuration defined
+at build time. There is a sample configuration file located at
+:file:`/usr/share/defaults telemetrics/telemetrics.conf` and represents the
```

Comment:
missing '/' in pathname.  Should be /usr/share/defaults/telemetrics/telemetrics.conf

Reply:
Fixed